cmd/cluster-network-check-endpoints/main.go

```diff
@@ -0,0 +1,16 @@
+package main
```

Comment:
It looks like you're referencing the image `cluster-kube-apiserver-operator` in the deployment; does this code need to be here?

Reply:
yeah, that was prior to pulling down the cluster-kube-apiserver-operator code for check-endpoints into CNO, it's a left over.
I guess I will need to decouple this cmd/cluster-network-check-endpoints into its own PR, land it, get it built, so then I can reference it in the deployment, I don't think I can self-reference on CI ?

Reply:
Filed https://github.com/openshift/cluster-network-operator/pull/872/files to decouple